spec/services/attribute_asserter_spec.rb

```diff
@@ -134,17 +49,25 @@
     )
   end
 
+  let(:subject) do
```

Comment:
this can be called just once, and we can update any changing values in the context blocks